src/components/forms/contactForm.jsx

```diff
@@ -92,11 +92,11 @@ class ContactForm extends Component{
         </form>
         <p></p>
 
-        <Button className="btn" color="primary" size="lg" onChange={this.handleSubmit}>
+        <Button className="btn" color="primary" size="lg" onChange={this.handleSubmit} href="/" >
```

Comment:
When we last spoke you said there was an issue with href not working properly once that POST axious call was added to the code. Do these work or are they not working as intended at the moment.

Reply:
The href works on the cancel button, however we have decided to remove it on the submit button. It appeared that it was not allowing the function calls to happen properly. In the master branch, we have removed the href from Submit, so this problem is fixed.